scripts/exrrfs_jedivar.sh

```diff
@@ -146,4 +146,4 @@ fi
 # the input/output file are linked from the umbrella directory, so no need to copy
 cp "${DATA}"/jdiag* "${COMOUT}/jedivar/${WGF}"
 cp "${DATA}"/jedivar*.yaml "${COMOUT}/jedivar/${WGF}"
-cp "${DATA}"/log.* "${COMOUT}/jedivar/${WGF}"
+cp "${DATA}"/log.out "${COMOUT}/jedivar/${WGF}"
```

Comment:
I think this works. But to confirm, we only need `log.out`? Thanks!

Reply:
Sorry, I just read your PR description on this. So it is good with me. Thanks!

Reply:
I think so. Thanks!